From c50ad182f8375eacb315b894229eec7f4cad1b30 Mon Sep 17 00:00:00 2001
From: Hugues KAMBA MPIANA <hugues.kambampiana@arm.com>
Date: Fri, 18 Oct 2024 11:50:32 +0000
Subject: [PATCH] smm-gateway/config: Remove `Werror` compile option

Adding `-Werror` to CMakeLists.txt unintended when
originally committed.

Remove the compilation option from `default-opteesp` and
`default-sp` configuration CMakeLists.txt files.

Upstream-Status: Backport [c50ad182f8375eacb315b894229eec7f4cad1b30]
Signed-off-by: Harsimran Singh Tungal <harsimransingh.tungal@arm.com>
Signed-off-by: Hugues KAMBA MPIANA <hugues.kambampiana@arm.com>
---
 deployments/smm-gateway/config/default-opteesp/CMakeLists.txt | 1 -
 deployments/smm-gateway/config/default-sp/CMakeLists.txt      | 1 -
 2 files changed, 2 deletions(-)

diff --git a/deployments/smm-gateway/config/default-opteesp/CMakeLists.txt b/deployments/smm-gateway/config/default-opteesp/CMakeLists.txt
index 94ff14c2..42385635 100644
--- a/deployments/smm-gateway/config/default-opteesp/CMakeLists.txt
+++ b/deployments/smm-gateway/config/default-opteesp/CMakeLists.txt
@@ -92,7 +92,6 @@ target_compile_definitions(smm-gateway PRIVATE
 if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
 	target_compile_options(smm-gateway PRIVATE
 		-std=c11
-		-Werror
 	)
 
 endif()
diff --git a/deployments/smm-gateway/config/default-sp/CMakeLists.txt b/deployments/smm-gateway/config/default-sp/CMakeLists.txt
index e92f1638..57802716 100644
--- a/deployments/smm-gateway/config/default-sp/CMakeLists.txt
+++ b/deployments/smm-gateway/config/default-sp/CMakeLists.txt
@@ -89,7 +89,6 @@ target_compile_definitions(smm-gateway PRIVATE
 if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
 	target_compile_options(smm-gateway PRIVATE
 		-std=c11
-		-Werror
 	)
 
 endif()
-- 
2.43.0

